python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -36,10 +36,9 @@
 from lsst.ip.diffim import ImagePsfMatchTask, DipoleAnalysis, \
     SourceFlagChecker, KernelCandidateF, cast_KernelCandidateF, makeKernelBasisList, \
     KernelCandidateQa, DiaCatalogSourceSelectorTask, DiaCatalogSourceSelectorConfig, \
-    GetCoaddAsTemplateTask, GetCalexpAsTemplateTask, DipoleFitTask
+    GetCoaddAsTemplateTask, GetCalexpAsTemplateTask, DipoleFitTask, ImageDecorrelationTask
```

Comment:
OK. I will rename it to DecorrelateALKernelTask, unless anyone can think of a better name.